Cargo.toml

```diff
@@ -48,6 +48,9 @@ tracing = { version = "0.1.41", default-features = false }
 bytemuck = "1.12.3"
```

Comment:
We'd still need someone to be the designated target maintainer in `.github/CODEOWNERS`.